node/podcontroller.go

```diff
@@ -115,6 +115,8 @@ type PodController struct {
 	// (derived from Kubernetes' cache library) -> a *knownPod struct.
 	knownPods sync.Map
 
+	eventFilterFunc func(interface{}) bool
```

Comment:
It seems like this should be defined as a type?

Reply:
Yep, updated.